<launch>

    <!--
    Args with no default attribute are not optional
    -->
    <arg name="config" default="$(find speechToTextPalbator)/config/config_v5.yaml" doc="Path to parameter config file"/>
    
    <!--
    Node for handling jsgf grammar mode
    -->
	<node
		name="SpeechRecognitionManager"
		pkg="speechToTextPalbator"
		type="SpeechRecognitionManager.py"
        output="screen"
        >
	    <rosparam file="$(arg config)"/>
	</node>


    
   
</launch>
